src/gravel/api/nfs.py

```diff
@@ -103,6 +112,28 @@ def get_service_info(service_id: str) -> NFSServiceModel:
                             detail=str(e))
 
 
+@router.post(
+    '/export/{service_id}',
```

Comment:
This is where you wanted to be able to do something like`@router.put('/export/{service_id}/{path}', ...)`, but couldn't because path is a path, and messes up the URL, right? Can you use https://fastapi.tiangolo.com/tutorial/path-params/#path-convertor here?

Reply:
Yes! So the question would be how do we prefer to represent the endpoints?

Using the export id:
`GET /nfs/export/<int>`

Using the psudo path:
`GET /nfs/export/foo/bar/baz`

Maybe we rework this in a follow-up PR?

Reply:
I think passing the path in the URL is bound to cause confusion. Passing it as a request is cleaner and tidier. :shrug: